dns/nixos.org.js

```diff
@@ -0,0 +1,186 @@
+D("nixos.org",
+	REG_NONE,
```

Comment:
What's this about?

Reply:
We don't interact with a registrar, so this is set to none.